railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -13,8 +13,10 @@ Rails.application.configure do
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
+  <%- unless options.skip_action_mailer? -%>
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
+  <%- end -%>
```

Comment:
This will leave 2 blank lines on the generated file, we can fix that by including one of them in the unless condition.